pkg/queue/manager.go

```diff
@@ -325,6 +331,7 @@ func (m *Manager) deleteWorkloadFromQueueAndClusterQueue(w *kueue.Workload, qKey
 		return
```

Comment:
unrelated to the PR, but can you unroll this function inside `DeleteWorkload`, seems unnecessary.

Reply:
It's also used in UpdateWorkload